pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -414,9 +479,8 @@ func (k *Deployer) getKptFnRunArgs() ([]string, error) {
 	}
 
 	count := 0
-
+	// fn-path is not supported due to kpt issue https://github.com/GoogleContainerTools/kpt/issues/1149
 	if len(k.Fn.FnPath) > 0 {
-		flags = append(flags, "--fn-path", k.Fn.FnPath)
```

Comment:
Its looks a little weird to not error out here. Can we error out with message "Config FnPath is not supported" ?
This could happen early on to fail fast when we validate schema

Reply:
I changed the comment. Skaffold should support fn-path. Previous release works because fn-path is used in a wrong way (it supposes to be refers to a directory, but used as specifying a certain file).